ixmp/backend/io.py

```diff
@@ -61,6 +62,9 @@ def s_write_excel(be, s, path, item_type):
     omitted = set()
     empty_sets = []
 
+    # Don't allow the user to set a value that will truncate data
+    max_row = int(min([x for x in [max_row, EXCEL_MAX_ROWS] if x is not None]))
```

Comment:
Is it just to calc min of 2?

Reply:
@zikolach, this is needed because the input argument may be None passed through other files (core.py and base.py).

Reply:
I see, just wondered if it can be expressed simpler ;)

Reply:
I modified as follows:
https://github.com/iiasa/ixmp/blob/96528d8c405e49e6da00d2f6afe5603b9e2b5fbb/ixmp/backend/io.py#L68-L69
1. If `max_rows` is None, then `max_rows or EXCEL_MAX_ROWS` evaluates to the latter.
2. Call `int()` on (1)
3. Then the `min()` of (2) and `EXCEL_MAX_ROWS`, i.e. possibly the min of two identical values.